sources/osg/Matrix.js

```diff
@@ -652,9 +653,9 @@ var Matrix = {
     postMultRotate: ( function () {
         var r = matrixCreate();
         return function ( matrix, quat ) {
-            Matrix.makeRotateFromQuat( r, quat );
+            Matrix.makeRotateFromQuat( quat, r );
             Matrix.postMult( matrix, r );
-            return matrix;
+            return r;
```

Comment:
So postmult(a,b) gives the same result as premult(a,b) ? The only thing that changes is the variable where you store the result? 

This is a bit confusing. 
Tomorrow I will review the maths. Thanks for your comments :+1: